scene/2d/animated_sprite.cpp

```diff
@@ -355,23 +355,32 @@ void AnimatedSprite::_notification(int p_what) {
 		case NOTIFICATION_DRAW: {
 
 			if (frames.is_null()) {
-				print_line("no draw no faemos");
+				print_line("no draw no faemos: " + String(get_path()));
```

Comment:
What is "faemos"?

Reply:
I have no idea what "faemos" is, I assume is Portuguese, but it is what was already there.